go.mod

```
@@ -0,0 +1,12 @@
+module github.com/microsoft/cobalt
```

Mod spec calls for case-sensitive module path here.
```suggestion
module github.com/Microsoft/cobalt
```

---

Our repository name is lower-cased so I was going to follow that convention with our module naming. Is there a convention or other reason to prefer upper case?

---

Should we include `*.mod` in our `.gitignore`?

---

> Should we include `*.mod` in our `.gitignore`?

Chatted with @nmiodice and agree with the rationale he took with including `go.mod` to version lock all dependencies